MapboxStatic/Overlay.swift

```diff
@@ -4,10 +4,10 @@
     import UIKit
 #endif
 
-let allowedCharacterSet: NSCharacterSet = {
-    let characterSet = NSCharacterSet.URLPathAllowedCharacterSet().mutableCopy() as! NSMutableCharacterSet
-    characterSet.removeCharactersInString("/)")
-    return characterSet
+let allowedCharacterSet: CharacterSet = {
+    let characterSet = (CharacterSet.urlPathAllowed as NSCharacterSet).mutableCopy() as! NSMutableCharacterSet
```

Comment:
Now we can mark this variable as `var` instead of copying and casting it. A `var` CharacterSet allows us to call `removeCharacters(in:)`, just like an NSMutableCharacterSet.